src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -191,6 +191,7 @@ def get_pairs(word):
     "ba": "bashkir",
     "jw": "javanese",
     "su": "sundanese",
+    "yue": "cantonese",
```

Comment:
Cantonese needs to be added here as well:
https://github.com/huggingface/transformers/blob/606d90845ff149ee08629edab95b576f788243a0/src/transformers/models/whisper/tokenization_whisper.py#L1209

It's not spaced, and was added to the unicode list here in the OAI Whisper repo: https://github.com/openai/whisper/blob/fcfeaf1b61994c071bba62da47d7846933576ac9/whisper/tokenizer.py#L278

Reply:
Good catch